src/ui/layer_list_panel.css

```diff
@@ -11,9 +11,24 @@
 .neuroglancer-layer-list-panel-item {
   display: flex;
   flex-direction: row;
-  padding: 2px;
+  padding: 2px 4px;
   border: 1px solid #aaa;
   margin: 2px;
+  align-items: center;
+  gap: 4px;
+}
+
+.neuroglancer-layer-list-panel-item input[type="checkbox"] {
+  width: 1rem;
+  height: 1rem;
+  margin: 4px;
```

Comment:
I think this margin is too much, might need to review design

Reply:
I see, in the design the checkbox itself is 16x16 and has margin of 4px and a gap between items is 4px: 
![image](https://github.com/user-attachments/assets/dfe3d98a-edca-422a-86f6-a9956d9d3f32)
but if it looks too much we can ask @syamace-metacell to reduce the size 
What do you think @seankmartin?